phys/module_pbl_driver.F

```diff
@@ -2204,6 +2214,25 @@ SUBROUTINE pbl_driver(                                          &
               ,ITS=its,ITE=ite,JTS=jts,JTE=jte,KTS=kts,KTE=kte)
           ENDIF
 
+     IF (PRESENT(ahe)) THEN
```

Comment:
@cenlinhe This is one place I am referring to. Use namelist to control the code, not the present of an array.